### PR TITLE
Websockets working!

### PR DESCRIPTION
**Pull request by kraleppa**

### CODE REVIEW

**Review by skupien — APPROVED** [review]
Great job resolving websocket problem

File: backend/src/main/kotlin/pl/edu/agh/dockerbuddy/config/WebSocketConfig.kt [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
-class WebSocketConfig : AbstractWebSocketMessageBrokerConfigurer() {
+class WebSocketConfig : WebSocketMessageBrokerConfigurer {
+    override fun configureMessageBroker(config: MessageBrokerRegistry) {
+        config.enableSimpleBroker("/alerts")
+    }
+
     override fun registerStompEndpoints(registry: StompEndpointRegistry) {
-        registry.addEndpoint("/ws")
+        registry.addEndpoint("/api/v2/ws")
+                //TODO it should not stay this way on prod
```
Comment (jakubsolecki) [inline backend/src/main/kotlin/pl/edu/agh/dockerbuddy/config/WebSocketConfig.kt:22]: oof

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -8,8 +8,20 @@ import { HostsDataProvider } from "./context/HostContext";
...
+import SockJS from "sockjs-client";
+import { Stomp } from "@stomp/stompjs";
 
 const App: React.FC = () => {
+  const socket = new SockJS("http://localhost:8080/api/v2/ws");
+  const stompClient = Stomp.over(socket);
+  stompClient.connect({}, function (frame: any) {
+    console.log("Connected: " + frame);
+    stompClient.subscribe("/alerts", function (greeting) {
+      console.log("AAAAAAAAA");
```
Comment (jakubsolecki) [inline frontend/src/App.tsx:20]: @kraleppa You can do better than this, even after 2 sleepless nights in a row

**Review by jakubsolecki — APPROVED** [review]

File: frontend/src/App.tsx (hunk 2 above)
Comment (kraleppa) [reply frontend/src/App.tsx:20]: I will kill myself soon, I promise

**Pull request merged**